postman/config.yml

```diff
@@ -0,0 +1,39 @@
+# Name of the quickstart (required)
+name: Postman
```

Comment:
```suggestion
name: postman
```

Reply:
Usually the naming convention here is lower case, the Title is what will be displayed on the page which will be capitalized.